tests/test_sphere.py

```diff
@@ -0,0 +1,41 @@
+import hashlib
+from pathlib import Path
+
+from cycax.cycad import Print3D
+from tests.shared import hex_code_check
+
+
+def sphere_cube(tmp_path: Path):
+    cube = Print3D(x_size=11, y_size=11, z_size=11, part_no="testing_sphere")
+
+    for side in (cube.left, cube.right, cube.top, cube.bottom, cube.front, cube.back):
```

Comment:
The previous two comments will make this quite nice.
```
from cycax.cycad.location import SIDES
for side_name on SIDES:
    side = cube.get_side(side_name)
    ...
```

Reply:
SS179